### PR TITLE
feat: add entities:list command

### PR DESCRIPTION
**Pull request by grof**

### CODE REVIEW

File: src/commands/entities/list.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    debug('setRequestActionMessage()')
+    this.requestActionMessage = `Retrieving entities for project ID ${options.project}`
+  }
+
+  transformResponse(result: MixResult) {
+    debug('transformResponse()')
+    const data = result.data as any
+
+    const entities = data?.entities.map((entity: any) => {
+      const [type] = Object.keys(entity)
```
Comment (oliver-daniel) [inline src/commands/entities/list.ts:95]: I left a similar comment in #20 regarding this logic of getting the first key, and then looking up the value. Is this operation "safe" in terms of ordering? I wonder if there's a more idiomatic way to handle this.

**Review by oliver-daniel — CHANGES_REQUESTED** [review]

File: src/mix/api/entities-types.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+/** @hidden */
+export type EntitiesListSearchParams = {
+  /** Entity type to filter the results */
+  type: string
+}
+
+/** @hidden */
+export type EntitiesGetParams = Expand<EntitiesGetPathParams>
+export type EntitiesListParams = Expand<EntitiesListPathParams & EntitiesListSearchParams>
```
[the remark this answers is not in the document]
Comment (grof) [reply src/mix/api/entities-types.ts:44]: Without the Expand, VSCode does not show useful information in Intellisense. Not a TypeScript pro but that worked.

**Review by oliver-daniel — APPROVED** [review]
Looks good.

**Pull request merged**